Makefile

```diff
@@ -15,11 +15,11 @@ VERSION?=$(shell cat VERSION | tr -d " \t\n\r")
 # The ldflags for the go build process to set the version related data.
 GO_BUILD_LDFLAGS=\
 	-s \
-	-X $(GO_PKG)/version.Revision=$(BUILD_REVISION)  \
```

Comment:
I tried to pass the variables to the config package, but it did not work. I tried on the database package and it worked, I'm could not figure out why it did not work on the config package so I left them on the main.